files/zh-cn/web/javascript/reference/global_objects/number/index.md

```diff
@@ -33,49 +36,49 @@ Number(undefined); // NaN
 
 ### Number 编码
 
-JavaScript 的 `Number` 类型是一个[双精度 64 位二进制格式 IEEE 754](https://zh.wikipedia.org/wiki/雙精度浮點數) 值，类似于 Java 或者 C# 中的 `double`。这意味着它可以表示小数值，但是存储数值的大小和精度有一些限制。简而言之，IEEE 754 双精度浮点数使用 64 位来表示 3 个部分：
+JavaScript 的 `Number` 类型是一个[双精度 64 位二进制格式 IEEE 754](https://zh.wikipedia.org/wiki/雙精度浮點數) 值，类似于 Java 或者 C# 中的 `double`。这意味着它可以表示小数值，但是存储的数字的大小和精度有一些限制。简而言之，IEEE 754 双精度浮点数使用 64 位来表示 3 个部分：
 
-- 1 比特*符号位*（sign）（正数或者负数）
-- 11 比特*指数位*（exponent）（-1022 到 1023）
-- 52 比特*尾数位*（mantissa）（表示 0 和 1 之间的数值）
+- 1 位用于表示*符号*（正数或者负数）
+- 11 位用于表示*指数*（-1022 到 1023）
+- 52 位用于表示*尾数*（表示 0 和 1 之间的数值）
```

Comment:
cloud we reserve the English annotates: sign, exponect and mantissa, as we have a mathematical formula below